Fix the following Node.js errors:

  [4511/75695] ACTION //chrome/browser/resources/gaia_auth_host:build_injected_scripts(//build/toolchain/linux/unbundle:default)
  FAILED: gen/chrome/browser/resources/gaia_auth_host/saml_injected.rollup.js gen/chrome/browser/resources/gaia_auth_host/build_injected_scripts_manifest.json
  python3 ../../ui/webui/resources/tools/minify_js.py --in_folder gen/chrome/browser/resources/gaia_auth_host/build_injected_scripts_bundled --out_folder gen/chrome/browser/resources/gaia_auth_host --out_manifest gen/chrome/browser/resources/gaia_auth_host/build_injected_scripts_manifest.json --in_files saml_injected.rollup.js
  Traceback (most recent call last):
    File "/tmp/chromium-136.0.7103.48/out/Release/../../ui/webui/resources/tools/minify_js.py", line 50, in <module>
      main(sys.argv[1:])
    File "/tmp/chromium-136.0.7103.48/out/Release/../../ui/webui/resources/tools/minify_js.py", line 35, in main
      node.RunNode([
    File "/tmp/chromium-136.0.7103.48/third_party/node/node.py", line 38, in RunNode
      raise RuntimeError('Command \'%s\' failed\n%s' % (' '.join(cmd), err))
  RuntimeError: Command '/tmp/chromium-136.0.7103.48/third_party/node/linux/node-linux-x64/bin/node /tmp/chromium-136.0.7103.48/out/Release/../../ui/webui/resources/tools/minify_js.js --in_folder /tmp/chromium-136.0.7103.48/out/Release/gen/chrome/browser/resources/gaia_auth_host/build_injected_scripts_bundled --out_folder /tmp/chromium-136.0.7103.48/out/Release/gen/chrome/browser/resources/gaia_auth_host saml_injected.rollup.js' failed
  (node:21678) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
  (Use `node --trace-warnings ...` to show where the warning was created)
  /tmp/chromium-136.0.7103.48/ui/webui/resources/tools/minify_js.js:11
  import assert from 'node:assert';
  ^^^^^^

  SyntaxError: Cannot use import statement outside a module
      at internalCompileFunction (node:internal/vm:74:18)
      at wrapSafe (node:internal/modules/cjs/loader:1141:20)
      at Module._compile (node:internal/modules/cjs/loader:1182:27)
      at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
      at Module.load (node:internal/modules/cjs/loader:1081:32)
      at Module._load (node:internal/modules/cjs/loader:922:12)
      at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
      at node:internal/main/run_main_module:23:47

  Node.js v18.13.0

----------------

  [21638/75695] ACTION //chrome/browser/resources/tts_engine:lint(//build/toolchain/linux/unbundle:default)
  FAILED: gen/chrome/browser/resources/tts_engine/eslint.config.mjs
  python3 ../../ui/webui/resources/tools/eslint_ts.py --in_folder gen/chrome/browser/resources/tts_engine/preprocessed --out_folder gen/chrome/browser/resources/tts_engine --config_base ../../../../../../../ui/webui/resources/tools/eslint_ts.config_base.mjs --tsconfig tsconfig_build_ts.json --in_files background.ts
  Traceback (most recent call last):
    File "/tmp/chromium-136.0.7103.48/out/Release/../../ui/webui/resources/tools/eslint_ts.py", line 79, in <module>
      main(sys.argv[1:])
    File "/tmp/chromium-136.0.7103.48/out/Release/../../ui/webui/resources/tools/eslint_ts.py", line 68, in main
      node.RunNode([
    File "/tmp/chromium-136.0.7103.48/third_party/node/node.py", line 38, in RunNode
      raise RuntimeError('Command \'%s\' failed\n%s' % (' '.join(cmd), err))
  RuntimeError: Command '/tmp/chromium-136.0.7103.48/third_party/node/linux/node-linux-x64/bin/node /tmp/chromium-136.0.7103.48/third_party/node/node_modules/eslint/bin/eslint --color --quiet --config gen/chrome/browser/resources/tts_engine/eslint.config.mjs gen/chrome/browser/resources/tts_engine/preprocessed/background.ts' failed
  (node:66369) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
  (Use `node --trace-warnings ...` to show where the warning was created)

  Oops! Something went wrong! :(

  ESLint: 9.20.1

  /tmp/chromium-136.0.7103.48/ui/webui/resources/tools/webui_eslint_plugin.js:5
  import {ESLintUtils} from '../../../../third_party/node/node_modules/@typescript-eslint/utils/dist/index.js';
  ^^^^^^

  SyntaxError: Cannot use import statement outside a module
      at internalCompileFunction (node:internal/vm:74:18)
      at wrapSafe (node:internal/modules/cjs/loader:1141:20)
      at Module._compile (node:internal/modules/cjs/loader:1182:27)
      at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
      at Module.load (node:internal/modules/cjs/loader:1081:32)
      at Module._load (node:internal/modules/cjs/loader:922:12)
      at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:169:29)
      at ModuleJob.run (node:internal/modules/esm/module_job:194:25)


diff --git a/ui/webui/resources/PRESUBMIT.py b/ui/webui/resources/PRESUBMIT.py
index 5bdc923fcf75..e5fa7bcfb3c7 100644
--- a/ui/webui/resources/PRESUBMIT.py
+++ b/ui/webui/resources/PRESUBMIT.py
@@ -106,7 +106,7 @@ def CheckNoDisallowedJS(input_api, output_api):
 def CheckNoNewPolymer(input_api, output_api):
   IGNORE_FILES = [
     # These files are needed for testing Polymer specific ESLint rules in
-    # ui/webui/resources/tools/webui_eslint_plugin.js.
+    # ui/webui/resources/tools/webui_eslint_plugin.mjs.
     'ui/webui/resources/tools/tests/eslint_ts/with_webui_plugin_polymer_property_class_member_violations.ts',
     'ui/webui/resources/tools/tests/eslint_ts/with_webui_plugin_polymer_violations.ts',
   ]
diff --git a/ui/webui/resources/tools/eslint_ts.config_base.mjs b/ui/webui/resources/tools/eslint_ts.config_base.mjs
index 3b4e7349230e..57c60cd741d9 100644
--- a/ui/webui/resources/tools/eslint_ts.config_base.mjs
+++ b/ui/webui/resources/tools/eslint_ts.config_base.mjs
@@ -4,7 +4,7 @@
 
 import typescriptEslint from '../../../../third_party/node/node_modules/@typescript-eslint/eslint-plugin/dist/index.js';
 import tsParser from '../../../../third_party/node/node_modules/@typescript-eslint/parser/dist/index.js';
-import webUiEslint from './webui_eslint_plugin.js';
+import webUiEslint from './webui_eslint_plugin.mjs';
 
 export default {
   languageOptions: {
diff --git a/ui/webui/resources/tools/eslint_ts.gni b/ui/webui/resources/tools/eslint_ts.gni
index 195bef145262..f46557f1cecf 100644
--- a/ui/webui/resources/tools/eslint_ts.gni
+++ b/ui/webui/resources/tools/eslint_ts.gni
@@ -27,7 +27,7 @@ template("eslint_ts") {
     inputs = [
       config_base,
       tsconfig,
-      "//ui/webui/resources/tools/webui_eslint_plugin.js",
+      "//ui/webui/resources/tools/webui_eslint_plugin.mjs",
     ]
     outputs = [ "$target_gen_dir/eslint.config.mjs" ]
 
diff --git a/ui/webui/resources/tools/minify_js.gni b/ui/webui/resources/tools/minify_js.gni
index 2b41102620ae..da1d39a1aa3f 100644
--- a/ui/webui/resources/tools/minify_js.gni
+++ b/ui/webui/resources/tools/minify_js.gni
@@ -15,7 +15,7 @@ template("minify_js") {
                              "out_folder",
                            ])
 
-    inputs = [ "//ui/webui/resources/tools/minify_js.js" ]
+    inputs = [ "//ui/webui/resources/tools/minify_js.mjs" ]
     outputs = []
     foreach(f, invoker.in_files) {
       assert(get_path_info(f, "extension") == "js")
diff --git a/ui/webui/resources/tools/minify_js.js b/ui/webui/resources/tools/minify_js.mjs
similarity index 100%
rename from ui/webui/resources/tools/minify_js.js
rename to ui/webui/resources/tools/minify_js.mjs
diff --git a/ui/webui/resources/tools/minify_js.py b/ui/webui/resources/tools/minify_js.py
index f442a877f70b..8f65814a70d5 100755
--- a/ui/webui/resources/tools/minify_js.py
+++ b/ui/webui/resources/tools/minify_js.py
@@ -33,7 +33,7 @@ def main(argv):
   # does not allow compressing multiple files at once. This is done to avoid
   # launching NodeJS once for every input file.
   node.RunNode([
-      os.path.join(_HERE_PATH, 'minify_js.js'), '--in_folder', in_path,
+      os.path.join(_HERE_PATH, 'minify_js.mjs'), '--in_folder', in_path,
       '--out_folder', out_path
   ] + args.in_files)
 
diff --git a/ui/webui/resources/tools/webui_eslint_plugin.js b/ui/webui/resources/tools/webui_eslint_plugin.mjs
similarity index 100%
rename from ui/webui/resources/tools/webui_eslint_plugin.js
rename to ui/webui/resources/tools/webui_eslint_plugin.mjs
